chainer/functions/loss/negative_sampling.py

```diff
@@ -16,6 +17,7 @@ def _sigmoid_grad(x, y, gy):
 class NegativeSamplingFunction(function_node.FunctionNode):
 
     ignore_label = -1
+    _samples = None
```

Comment:
Do we need to change the attribute name from `samples` to `_samples`?

Reply:
Reverted to `samples`.